fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/ParameterHashVisitor.java

```diff
@@ -57,6 +57,18 @@ public ParameterHashVisitor() {
         encoder = Base64.getEncoder();
     }
 
+    /**
+     * If legacyWholeSystemSearchParamsEnabled is true, then update digest to ensure hash changes
+     * from when legacyWholeSystemSearchParamsEnabled is false; this can be removed when the
+     * legacyWholeSystemSearchParamsEnabled config setting is removed.
+     * @param enabled true or false
+     */
+    public void setLegacyWholeSystemSearchParamsEnabled(boolean enabled) {
+        if (enabled) {
+            updateDigestWithString("legacyWholeSystemSearchParamsEnabled");
+        }
+    }
```

Comment:
any. thoughts on adding this to the constructor? 
design reasons?

Reply:
would this be a pre-fixed setting. option?

Reply:
Yeah, it could be added to the constructor instead by adding a 2nd constructor that takes the parameter to avoid having to change a bunch of testcases.

Reply:
Updated.

Reply:
> would this be a pre-fixed setting. option?

Is what I have ok?  Not sure I understand this question.